x/authz/spec/03_messages.md

```diff
@@ -4,37 +4,40 @@ order: 3
 
 # Messages
 
-In this section we describe the processing of messages for the authz module.
+In this section we describe the Msg Protobuf service for the authz module.
 
-## Msg/Grant
+## Grant
 
 An authorization-grant is created using the `MsgGrant` message.
 
-+++ https://github.com/cosmos/cosmos-sdk/blob/c95de9c4177442dee4c69d96917efc955b5d19d9/proto/cosmos/authz/v1beta1/tx.proto#L27-L35
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-beta1/proto/cosmos/authz/v1beta1/tx.proto#L33-L37
 
 This message is expected to fail if:
 
 - both granter & grantee have same address.
 - provided `Expiration` time less than current unix timestamp.
 - provided `Authorization` is not implemented.
+-
 - Authorization Method doesn't exist (there is no defined handler in the app router to handle that Msg types)
 
-## Msg/Revoke
+If there is already a grant for `(granter, grantee, AuthorizationType)` triple, then the new grant will overwrite the previous one. To update or extend an existing grant a new grant with same `(granter, grantee, AuthorizationType)` should be created.
+
+## Revoke
 
 An allowed authorization can be removed with `MsgRevoke` message.
 
-+++ https://github.com/cosmos/cosmos-sdk/blob/c95de9c4177442dee4c69d96917efc955b5d19d9/proto/cosmos/authz/v1beta1/tx.proto#L53-L59
++++ https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-beta1/proto/cosmos/authz/v1beta1/tx.proto#L60-L64
```

Comment:
```suggestion
+++ https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-beta1/proto/cosmos/authz/v1beta1/tx.proto#L58-L64
```

Reply:
This does not need to be changed. Maybe this is where you meant to respond with "ditto". Resolving the suggestion.